src/components/expand/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import '~styles/settings';
+
+$z-index-blender: 2;
+$z-index-expand-content: 3;
+$button-width: 430px;
+$button-height: 70px;
+
+.c-expand {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  overflow-y: hidden;
+
+  .blender {
+    position: absolute;
+    left: calc(50% - #{$button-height / 2});
+    bottom: 1rem;
+
+    height: $button-height;
+    width: $button-height;
+    margin: auto;
+    border-radius: 50%;
+    background: white;
+    transition: transform 1s ease-in-out;
+    // mix-blend-mode: difference;
+    z-index: $z-index-blender;
+  }
+
+  .expand {
+    transform: scale(100);
+  }
+
+  .expand-content {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 1s ease-in-out;
+    z-index: $z-index-expand-content;
+
+    &.visible {
```

Comment:
please use correct notation

Reply:
I think you meant this comment. Lets use either bem or the vizzuality's notation (dont know the name), in which case modifiers should start with a dash: `-visible`